plugins/AIAgents/API.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\AIAgents;
+
+use Piwik\API\Request;
+use Piwik\DataTable;
+use Piwik\DataTable\DataTableInterface;
+use Piwik\Period;
+use Piwik\Piwik;
+use Piwik\Plugin\API as PluginAPI;
+use Piwik\Plugins\API\DataTable\MergeDataTables;
+use Piwik\Segment;
+use Piwik\Segment\SegmentExpression;
+use Piwik\Site;
+
+class API extends PluginAPI
+{
+    public const AI_AGENT_SEGMENT       = 'aiAgentName!=';
+    public const AI_AGENT_COLUMN_SUFFIX = '_ai_agent';
+
+    public const HUMAN_SEGMENT       = 'aiAgentName==';
+    public const HUMAN_COLUMN_SUFFIX = '_human';
+
+    /**
+     * @param int|string $idSite
+     * @param string $segment
+     * @param bool|string|array<string> $columns
+     */
+    public function get(
+        $idSite,
+        string $period,
+        string $date,
+        string $segment = '',
+        $columns = false
```

Comment:
we could maybe use an empty string or empty array as default here. Should have the same effect, but removes the requirement to have bool in the type hint